SimMuon/GEMDigitizer/src/GEMSimpleModel.cc

```diff
@@ -216,7 +218,10 @@ void GEMSimpleModel::simulateNoise(const GEMEtaPartition* roll, CLHEP::HepRandom
                                       + GE11ElecBkgParam1 * rollRadius
                                       + GE11ElecBkgParam2 * rollRadius * rollRadius
                                       + GE11ElecBkgParam3 * rollRadius * rollRadius * rollRadius;
+      
+    // Scale up/down for desired instantaneous lumi (reference is 5E34, double from config is in units of 1E34)
     averageNoiseRatePerRoll = averageNeutralNoiseRatePerRoll + averageNoiseElectronRatePerRoll;
+    averageNoiseRatePerRoll *= instLumi_*rateFact_*1.0/5;
```

Comment:
"magic number" 5(E34) should be made into a Python parameter in `muonGEMDigis_cfi.py`